e2e/test/scenarios/onboarding/command-palette.cy.spec.js

```diff
@@ -247,4 +247,14 @@ describe("command palette", () => {
       cy.location("pathname").should("eq", "/metric/query");
     });
   });
+
+  it("should show the 'File a bug' command palette item", () => {
```

Comment:
The functionality here would already be covered in https://github.com/metabase/metabase/blob/49585-introduce-a-cmdctrl--k-command-called-report-bug/e2e/test/scenarios/admin-2/error-reporting.cy.spec.ts#L48. Can we update this test to reflect the new name?